src/index.js

```diff
@@ -87,9 +128,15 @@ class ProfilerPlugin {
     return response.signedRequest;
   }
 
-  postInvoke() {
+  async postInvoke() {
     if (!this.enabled) return false;
 
+    try {
+      await this.pluginReadyPromise;
+    } catch (err) {
+      return false;
```

Comment:
This could be a smart place to put in debugger output

Reply:
Sounds good - I'll do that